test/python/pybind_test.py

```diff
@@ -140,6 +165,8 @@ def test_pybind_constant_size_vector_of_vector_return(self):
         self.assertEqual(vector_return.shape, (10, 10))
         self.assertEqual(np.sum(vector_return), 550)
 
+    #TODO test examples in python_bindings directory (currently includes pagerank_delta, sssp) 
```

Comment:
Are we going to fix this TODO?

Reply:
Ah this should be gone! i added the tests above. I was thinking about testing their python binding code but i figured it makes more sense to test the graphit code.